Makefile

```diff
@@ -300,6 +300,14 @@ $(ISO_BUILD_IMAGE): deploy/iso/minikube-iso/Dockerfile
 	@echo ""
 	@echo "$(@) successfully built"
 
+storage-provisioner-main: cmd/storage-provisioner/main.go
+	go build cmd/storage-provisioner/main.go
```

Comment:
rename to `storage-provisioner` and can you make this output to our build dir? `go build ... -o $(BUILD_DIR)/storage-provisioner`